web-components/src/global.css.js

```diff
@@ -237,6 +246,39 @@ merch-card [slot='heading-xl'] {
     color: var(--merch-color-grey-80);
 }
 
+merch-card [slot='callout-text'] {
+    display: inline-block;
+    margin: var(--consonant-merch-spacing-xxxs) 0px;
+}
+
+.callout-content-wrapper {
+    position: relative;
+    display: inline-grid;
+    background: rgba(203 203 203 / 50%);
```

Comment:
nit: use `var` for background color.

Reply:
it is not letting me override the opacity for the text, hence had to use rgba 
https://stackoverflow.com/questions/806000/how-do-i-reduce-the-opacity-of-an-elements-background-using-css